app/Http/Controllers/Api/V1/BoqItem/BoqItemController.php

```
@@ -20,9 +20,9 @@ public function index(Request $request)
     {
         if ($request->has('phase_id')) {
             $phase = BoqPart::find($request->phase_id);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\BoqPart' in method 'index'. (Clean Code Rules)

(StaticAccess)